server/platform/services/sharedchannel/sync_send_remote.go

```diff
@@ -679,6 +680,65 @@ func (scs *Service) sendProfileImageSyncData(sd *syncData) {
 }
 
 // sendSyncMsgToRemote synchronously sends the sync message to the remote cluster (or plugin).
```

Comment:
nit: this comment line needs to be moved to like 741 right before the sendSyncMsgToRemote function